whole-tale/girder_wholetale
# Manifest should only expand subfolders in dataSet. Fixes #439

Do not expand root dataset folder in the manifest.

### How to test?
1. Deploy alongside with https://github.com/whole-tale/wt_versioning/pull/13
1. Follow the steps from #439 and confirm that dataset before export == dataset after import.
